rosdep/python.yaml

```diff
@@ -5057,6 +5057,10 @@ python3-cairo:
     slackpkg:
       packages: [py3cairo]
   ubuntu: [python3-cairo]
+python3-can:
+  debian: [python3-can]
+  fedora: [python3-can]
+  ubuntu: [python3-can]
```

Comment:
Unfortunately, while this package is available on Ubuntu Bionic forwards, we also want to maintain backwards compatibility with Xenial.  So I think we'll need change back to the wildcard, but nulling out xenial, something like:

```
python3-can:
  debian: [python3-can]
  fedora: [python3-can]
  ubuntu:
    '*': [python3-can]
    xenial: null
```

Reply:
Okay, did that.